hs_app_timeseries/templates/pages/timeseriesresource.html

```diff
@@ -37,7 +37,9 @@
                         <table class="custom-table">
                             <tbody>
                                 <tr><th class="text-muted">Code </th><td>{{ site.site_code }}</td></tr>
-                                <tr><th class="text-muted">Name </th><td>{{ site.site_name }}</td></tr>
+                                {% if site.site_name %}
+                                    <tr><th class="text-muted">Name </th><td>{{ site.site_name }}</td></tr>
```

Comment:
do we have guidelines for this type of formatting in a template? I'm ok with this, but it's worth noting the extra indentation is rendered in the html.